content/wallets/index.zh.md

```diff
@@ -0,0 +1,155 @@
+        platformImg1Src="/images/platforms/android.png"
+        platformImg2Src="/images/platforms/mac.png"
+        platformImg3Src="/images/platforms/windows.png"
+        platformImg4Src="/images/platforms/linux.png"
+        platformImg5Src="/images/platforms/pwa.png"
+        platformImg6Src="/images/platforms/web.png"
+        newTab="正确的"
+    >}}
+    {{< wallet
+        titleText="NEXT"
+        walletImgSrc="/images/wallets/wal-next.png"
+        text="一个现代和时尚的NEXT钱包NavCoin。除了加注外，NEXT还提供了一个简单的界面，用于创建和投票社区资金&nbsp;提案。"
+        buttonUrl="http://next.navcommunity.net/"
+        buttonText="下载"
+        buttonImgSrc="/images/icons/rightward-arrow.svg"
+        platformImg1Src="/images/platforms/mac.png"
+        platformImg2Src="/images/platforms/windows.png"
+        platformImg3Src="/images/platforms/linux.png"
+        newTab="正确的"
+    >}}
+    {{< wallet
+        titleText="Coinomi"
+        walletImgSrc="/images/wallets/wal-coinomi.png"
+        text="Coinomi是一个安全第一的多重资产钱包，用于比特币、替代币和&nbsp;代币."
+        buttonUrl="https://www.coinomi.com/"
+        buttonText="下载"
+        buttonImgSrc="/images/icons/rightward-arrow.svg"
+        platformImg1Src="/images/platforms/mac.png"
+        platformImg2Src="/images/platforms/android.png"
+        newTab="正确的"
+    >}}
+    {{< wallet
+        titleText="NavPi StakeBox"
+        walletImgSrc="/images/wallets/wal-pi.png"
+        text="基于树莓派平台的低能耗NavCoin full&nbsp;钱包。"
+        buttonUrl="https://navhub.org/projects/nav-pi/"
+        buttonText="下载"
+        platformImg1Src="/images/platforms/pi.png"
+        buttonImgSrc="/images/icons/rightward-arrow.svg"
+        newTab="正确的"
+    >}}
+    {{< wallet
+        titleText="NavCoin M8"
+        walletImgSrc="/images/wallets/wal-m8.png"
+        text="一个高性能的NavCoin加注钱包，包括 NEXT钱包与diy&c&nbsp;选项。"
+        buttonUrl="https://navcoinm8.com/"
+        buttonText="了解更多"
+        buttonImgSrc="/images/icons/rightward-arrow.svg"
+        platformImg1Src="/images/platforms/linux.png"
+        newTab="正确的"
+    >}}
+    {{< wallet
+        titleText="NavPaper"
+        walletImgSrc="/images/wallets/wal-paper.png"
+        text="生成要打印和存储的密钥以提高&nbsp;安全性。"
+        buttonUrl="https://paper.navcoin.org/"
+        buttonText="下载"
+        buttonImgSrc="/images/icons/rightward-arrow.svg"
+        platformImg1Src="/images/platforms/mac.png"
+        platformImg2Src="/images/platforms/windows.png"
+        platformImg3Src="/images/platforms/web.png"
+        newTab="正确的"
+    >}}
+{{< /item_container_section >}}
+
+{{< item_container_section
+    titleText="下载NavCoin Core 4.5.0"
+    subtitleText="NavCoin Core钱包经过精心提炼，直观易用。它附带了一个安装向导，以及加载区块链并开始在安全环境中发送交易所需的&nbsp;一切。"
+    bgPurple="正确的"
+    anchor="download-core"
+    buttonUrl="https://github.com/NAVCoin/navcoin-core/releases/latest"
+    buttonText="先进的下载"
+    buttonImgSrc="/images/icons/rightward-arrow.svg"
+    newTab="正确的"
+>}}
+    {{< action
+        titleText="窗口"
```

Comment:
This is Windows the operating system, not an actual window. Translation is incorrect

Reply:
fixed